src/owner.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable class-methods-use-this */
+/* disabled for this because there's no need to reference .this */
+
+import {ethers} from "ethers";
+import {EthereumAddress, EthereumTransactionHash} from "./types";
+import {getWeb3Provider, getWallet} from "./provider";
+import {getLogger} from "./util/logger";
+import {abi as TitleEscrowABI} from "../build/contracts/TitleEscrow.json";
+
+const {error} = getLogger("owner");
+
+const NUMBER_OF_CONFIRMATIONS = 1; // number of confirmations we should wait before declaring a transaction succeeded
+
+export const isAddressTitleEscrow = async ({
+  address,
+  web3Provider = getWeb3Provider()
+}: {
+  address: EthereumAddress;
+  web3Provider?: ethers.providers.BaseProvider;
+}) => {
+  try {
+    const MaybeTitleEscrowContract = new ethers.Contract(address, JSON.stringify(TitleEscrowABI), web3Provider);
+    return await MaybeTitleEscrowContract.supportsInterface("0xad3fae94");
+  } catch (e) {
+    error(e);
+  }
+  return false;
+};
+
+export abstract class Owner {
```

Comment:
I think dont need to export it via lib. within lib is okay.  we export * from index.ts.